server/agents/explicit_meta_agent.py

```diff
@@ -20,7 +20,7 @@
 <Transcript start>{conversation_context}<Transcript end>
 
 [Your Task]
-Now use your tools and knowledge to answer the query to the best of your ability. The question or request you are to answer is the last (final) question/request posed by the human to you in the below 'Query'. Make your answer as concise and succinct as possible. Leave out filler words and redundancy to make the answer high entropy and as to-the-point as possible. Never answer with more than 240 characters, and try to make it even less than that. Most answers can be given in under 10 words.
+Now use your knowledge and/or tools (if needed) to answer the query to the best of your ability. Do not use your tools if you already know the answer to the query. The query may accidentally contain some extra speech at the end, you should ignore any noise and try to find the user's inteded query. Make your answer as concise and succinct as possible. Leave out filler words and redundancy to make the answer high entropy and as to-the-point as possible. Never answer with more than 240 characters, and try to make it even less than that. Most answers can be given in under 10 words.
```

Comment:
line 38 below, I was wondering if we could move the new_expert_agent out of the function and define it somewhere in the file instead, I think it is fine to reuse the same generic agent instead of creating one everytime the explicit agent is called